composer.json

```diff
@@ -11,9 +11,9 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
-        "puli/composer-plugin": "1.0.0-beta8",
-        "puli/discovery": "1.0.0-beta8"
+        "php": "^5.4|7.*",
```

Comment:
its semantically the same. maybe caret is better for when we need to bump
to some php 7 that includes a fix (if we run into that situation). but we
can also change once it becomes relevant